ansible/roles/schulcloud-server-tspsync/tasks/main.yml

```diff
@@ -111,3 +111,24 @@
       state: "{{ 'present' if WITH_TSP_SYNC|bool else 'absent'}}"
     tags:
       - configmap
+
+  - name: remove old API TSP Sync Init Job
```

Comment:
Do you realy want to remove the API TSP Sync Init Job every time?

Reply:
Without this removal the deployment failed due to it trying to update the job if it already existed

Reply:
Okay, remove this ansible task and if you need the job rerun then delete it manuel at the cluster.
Other wise every playbock run this job will be removed and recreated

Reply:
Added a mechanism like for the erwin init job to run it only once and added the required removal for not WITH_TSP_SYNC